spec/system/submissions/review_spec.rb

```diff
@@ -332,11 +332,20 @@
 
       within("div[data-checklist-item='1']") do
         within("div[data-result-item='0']") do
+          find("button[title='Move Down checklist result']").click
+        end
+        within("div[data-result-item='1']") do
+          find("button[title='Move Up checklist result']").click
```

Comment:
This just checks for the presence of the move-down button. The spec should check if the button is actually working.

Reply:
Fixed.